### PR TITLE
improved readability, ready for new languages additions

### PR DESCRIPTION
**Pull request by Roxe322**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -51,54 +64,30 @@ def get_confirm_message(lang: str):
...
-        return {
-            'text': get_confirm_message(GER),
-        }
-
-    if r < 10:
-        return {
-            'text': CONFIRMS[RUS]['hoot']
-        }
-    return {}
+    for name in AGREE_NAMES.keys():
```
Comment (AlwxSin) [inline bot.py:69]: Too many loops for me.
For more readability I think there is better to make functions for checking `is_reply_needed`, for languange detection etc.

Comment (AlwxSin) [reply bot.py:69]: Will you do that please?

**Review by AlwxSin — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (Roxe322) [reply bot.py:69]: Updated

File: bot.py [hunk 2]
```diff
@@ -50,55 +64,41 @@ def get_confirm_message(lang: str):
     return CONFIRMS[lang][key]
 
 
+def detect_language(text: str) -> Optional[str]:
+    language = None
+    for name in AGREE_NAMES.keys():
+        if name in text:
+            language = AGREE_NAMES[name]
```
Comment (AlwxSin) [inline bot.py:71]: If we found language, we should immediately return. No need to finish the loop.

File: bot.py [hunk 3]
```diff
@@ -50,55 +64,41 @@ def get_confirm_message(lang: str):
...
+    return language
+
+
+def check_is_reply_needed(text: str) -> bool:
+    is_reply_needed = False
+    for owl_name in OWL_NAMES.keys():
+        if owl_name in text:
+            is_reply_needed = True
```
Comment (AlwxSin) [inline bot.py:80]: Same here. If we found needed result no need to finish the loop.

**Review by AlwxSin — CHANGES_REQUESTED** [review]

File: bot.py (hunk 2 above)
Comment (Roxe322) [reply bot.py:71]: Updated

File: bot.py (hunk 3 above)
Comment (Roxe322) [reply bot.py:80]: Also updated

**Pull request merged**